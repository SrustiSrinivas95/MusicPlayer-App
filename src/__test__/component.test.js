import React from 'react';
import {shallow} from 'enzyme';
import Input from '../components/Atoms/Input';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('Input', () => {    
    let wrapper;
   beforeEach(() => {
    wrapper = shallow(<Input />)                                
  });
    it('Is Input component renders', () => {
      expect(wrapper).toMatchSnapshot();
    });
});