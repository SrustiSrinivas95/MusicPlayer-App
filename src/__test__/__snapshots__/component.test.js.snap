// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Is Input component renders 1`] = `ShallowWrapper {}`;
